Dockerfile

```diff
@@ -6,9 +6,9 @@ COPY package*.json ./
 
 ARG API_URL
 
-ENV API_URL ${API_URL}
+ENV API_URL = ${API_URL}
```

Comment:
I don't think the = is correct syntax based on what it was before

Reply:
Yeah just removed the url and the equals sign :( thats it fixed